src/Symfony/Bundle/WebProfilerBundle/EventListener/WebDebugToolbarListener.php

```diff
@@ -115,6 +117,27 @@ public function onKernelResponse(ResponseEvent $event): void
             $response->headers->remove('Location');
         }
 
+        if ($response->headers->has('X-Debug-Token') && $response instanceof EventStreamResponse) {
+            $callback = $response->getCallback();
+            $response->setCallback(static function () use ($callback, $response) {
+                $response->sendEvent(new ServerEvent(
+                    [
+                        $response->headers->get('X-Debug-Token') ?? '',
```

Comment:
this data is static, so why repeat it for every message?

Reply:
Hmm what do you mean? It's only included once per stream/request `$callback()` outputs all the messages user `yield`ed

Reply:
got it sorry!